### ISSUE TITLE
Amount entered into Send fungible (native/ERC 20) screen is not taken as it is

### ISSUE THREAD

**Issue by hboon** [AlphaWallet/alpha-wallet-ios#2270]
Amount entered into Send fungible (native/ERC 20) screen is not taken as it is

Maybe it's a bug introduced in #2249 ?

This also cause the problem where the user sees and knows there is a balance of "0.1" but is not allowed to enter "0.1"

(screenshot is taken with actionsheet PR, but issue is reproducible on master)

<img width="387" alt="Screenshot 2020-11-12 at 2 26 38 PM" src="https://user-images.githubusercontent.com/56189/98904038-c83c9b00-24f3-11eb-91bf-72c7da2e193f.png">

**Comment by vladyslav-iosdev** [comment]
@hboon i need to start on it after #2105 is merged?

**Comment by hboon** [comment]
@vladyslav-iosdev no, you can reproduce it in `master`. So you can fix it right away.

**Comment by vladyslav-iosdev** [comment]
@hboon i am not able to reproduce this, i tried with different amount and different locales, and all the time get right value.

**Comment by hboon** [comment]
Hmm. I thought I could reproduce it. I’ll check again. Thanks

**Comment by hboon** [comment]
Strange. On both master (b785a5fcfa1cdd22495cd42ac3f3d5f8edb9346f) as well as #2105, I can reproduce this. Going to revert the #2249 on master so as to merge actionsheet in and make a fortnight build first. We can come back and fix the original toggling problem that #2249 was addressing.

**Comment by vladyslav-iosdev** [comment]
@hboon i send another PR based on #2243 please let me know if it still reproducible for u
